code/modules/projectiles/guns/projectile/constructable/siegecannon.dm

```diff
@@ -487,7 +487,7 @@
 	desc = "A large sphere of honk."
 	icon = 'icons/obj/siege_cannon.dmi'
 	icon_state = "clownnonball"
-	starting_materials = list(MAT_CLOWN = CC_PER_SHEET_METAL*3)
+	starting_materials = list(MAT_CLOWN = CC_PER_SHEET_METAL*10.7) //20.006 bananium sheets
```

Comment:
```suggestion
	starting_materials = list(MAT_CLOWN = CC_PER_SHEET_CLOWN*20)
```
Begone, magic number! 
I don't see any specific reason to use CC_PER_SHEET_METAL here (crafting recipe seems to be ok). And if there is I'd argue that should be fixed first, rather than condensing a clown-to-metal ratio into some magic number no one will remember about if that ratio changes some day.

Reply:
Ah, I did not know there was a CC_per_sheet_clown.